### ISSUE TITLE
Create Build command for deployment

### ISSUE THREAD

**Issue by deepakgd** [angular/angular2-seed#46]
Like angular/angular-cli can you create dist folder automatically for deployment purpose

**Comment by skovmand** [comment]
+1 Preferably including minification

**Comment by gdi2290** [comment]
There's (beta.1-beta.9) currently a problem with minification. I can see if there is a setup that makes sense without introducing complexity

**Comment by deepakgd** [comment]
ya fine do mangle: false to avoid minification issue now.  Good solution need to find soon

**Comment by skovmand** [comment]
@gdi2290 Okay.. I'm actually glad to hear there's still a minification issue in beta 9, since I have been struggling with it. It's not only me, then. I just rely on gzipping for now. No minification at all, but looking forward to reducing the bundle size!

**Comment by zachgrayio** [comment]
This is actually pretty easy to do on your own, assuming you've got this in your webpack config:

**Comment by zachgrayio** [comment]
And as far as minification goes, so far it seems like `webpack.optimize.UglifyJsPlugin()` with minimize=true seems to be working out pretty well on ng2 beta.13, so that's worth a shot if you're adding a build task.

**Comment by piotrromanowski** [comment]
@zachgrayio how did you get the component html templates to load in dist? when opening the index.html I receive `Failed to load resouce: the server responded with a status 404 (Not Found) app/seed-app.html`

**Comment by pkozlowski-opensource** [comment]
Duplicate of #17. I think it will make sense to do this only once offline compilation works.
